project-api/project-api-test/src/test/resources/projects_for_resolution_tests/package_with__http_plugin_compile_issue_in_tests/tests/test.bal

```diff
@@ -0,0 +1,7 @@
+import ballerina/http;
```

Comment:
does this get resolved? Shall we use a different org name to be on the safe side?

Reply:
Removed this exact case and added a test by adding a custom code analyzer plugin.